Kotlin/lib/src/main/java/com/wolt/blurhashkt/Base83.kt

```diff
@@ -0,0 +1,34 @@
+package com.wolt.blurhashkt
+
+internal object Base83 {
+
+    private val CHAR_MAP = listOf(
+            '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F', 'G',
+            'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X',
+            'Y', 'Z', 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o',
+            'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', '#', '$', '%', '*', '+', ',',
+            '-', '.', ':', ';', '=', '?', '@', '[', ']', '^', '_', '{', '|', '}', '~'
+    )
+
+	fun encode(value: Int, length: Int, buffer: CharArray, offset: Int) {
```

Comment:
There's a white space here in front of the function. Also can you add a line feed to this file? :pray:

Reply:
Fixed